package/MDAnalysis/core/universe.py

```diff
@@ -168,6 +168,14 @@ class Universe(object):
         [``None``] Can also pass a subclass of
         :class:`MDAnalysis.coordinates.base.ProtoReader` to define a custom
         reader to be used on the trajectory file.
+    all_coordinates : bool
+        If set to ``True`` specifies that if more than one filename is passed
+        they are all to be used, if possible, as coordinate files (employing a
+        :class:`MDAnalysis.coordinates.chain.ChainReader`). [``False``] The
+        default behavior is to take the first file as a topology and the
+        remaining as coordinates. The first argument will always always be used
+        to infer a topology regardless of *all_coordinates*. This parameter is
```

Comment:
This is a little confusing, but I suppose it makes sense. Basically, even if you activate `all_coordinates`, the first specified file will tentatively be used both to generate a minimal topology AND as a coordinate file.